kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/BatchesResourceSuite.scala

```diff
@@ -79,9 +105,18 @@ class BatchesResourceSuite extends KyuubiFunSuite with RestFrontendTestHelper wi
     val response = webTarget.path("api/v1/batches")
       .request(MediaType.APPLICATION_JSON_TYPE)
       .post(Entity.entity(requestObj, MediaType.APPLICATION_JSON_TYPE))
-    assert(200 == response.getStatus)
+    assert(response.getStatus === 200)
```

Comment:
minor: I would prefer `assertResult` rather than `assert`. more clear in expected results.

Reply:
I know your point, but I suppose `assert(result === expected)` is generic too, it is used widely in scalatest.

Reply:
Yes, you're right. Although putting the object in the first place raises NPE concerns. LGTM.